test/button_test.js

```diff
@@ -6,6 +6,7 @@ const clipboardy = require("clipboardy");
 const firefox = require("selenium-webdriver/firefox");
 
 const Context = firefox.Context;
+const MAX_TIMES_TO_SHOW = 5; // this must match MAX_TIMES_TO_SHOW in bootstrap.js
```

Comment:
Per our in person discussion, because `button_test.js` is a Node file, importing `Config.jsm` will be more trouble than it's worth.